components/collator/src/lib.rs

```diff
@@ -285,12 +322,13 @@ extern crate alloc;
 pub use comparison::Collator;
 pub use comparison::CollatorBorrowed;
 pub use comparison::CollatorPreferences;
+pub use icu_locale_core::preferences::extensions::unicode::keywords::CollationCaseFirst as CaseFirst;
+pub use icu_locale_core::preferences::extensions::unicode::keywords::CollationNumericOrdering as NumericOrdering;
+pub use icu_locale_core::preferences::extensions::unicode::keywords::CollationType;
 pub use options::AlternateHandling;
```

Comment:
Observation: in icu_datetime, `options` is a public module/namespace. But icu_collator re-exports the things from the top level.

Reply:
https://github.com/unicode-org/icu4x/issues/5991